build_tools/github_actions/ci_build_bazel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+# Copyright 2023 The StableHLO Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if [[ $# -ne 0 ]] ; then
+  echo "Usage: $0"
+  exit 1
+fi
+
+# Build StableHLO
+echo "=== Building StableHLO ==="
+bazel build //:all
```

Comment:
Does this do the same thing as `bazel build //...`

Reply:
Yes indeed